packages/core/src/renderers/systems/filter/FilterSystem.js

```diff
@@ -153,6 +159,18 @@ export default class FilterSystem extends WebGLSystem
         globalUniforms.destinationFrame = state.destinationFrame;
         globalUniforms.resolution = state.resolution;
 
+        globalUniforms.filterArea[0] = state.destinationFrame.width;
```

Comment:
wander if we should we stick an if legacy around these?

Reply:
I dont know how to add if here, its about multiple filters at once.

Reply:
Also there'll be more parameters here anyway.